packages/quantic/cypress/integration/facets/timeframe-facet/timeframe-facet.cypress.ts

```diff
@@ -21,13 +21,13 @@ describe('quantic-timeframe-facet', () => {
   const pageUrl = 's/quantic-timeframe-facet';
 
   const validRange = {
-    start: '2000-01-01',
-    end: '2000-12-31',
+    start: '1/1/2000',
```

Comment:
Maybe it's the timezone on the CI vs locally?

Reply:
I'm thinking the locale would be the difference yes given that's the only parameter we pass to the salesforce utility for formatting. Kind of annoying.

Reply:
I'll revert that part of the changeset so we can get the build unblocked and investigate on the side.

Reply:
I made an addition to the github actions script to use an action from the marketplace to set the timezone. Let's see.

Reply:
Damn it looks like it's against the rules.

Reply:
Does this work ? https://github.com/cypress-io/cypress/issues/1043#issuecomment-623293020